regression/ebmc/SVA-LTL/lasso1-small-bound.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
+lasso1.sv
+--bound 10
+^EXIT=0$
+^SIGNAL=0$
+^\[top\.property\.p0\] .* SUCCESS$
```

Comment:
Why do we expect the property to pass? Both the source code and the other .desc file suggest that verification should fail.

Reply:
explanation added